ui/src/kapacitor/components/AlertTabs.js

```diff
@@ -228,11 +227,31 @@ class AlertTabs extends Component {
 
         <Tabs tabContentsClass="config-endpoint">
           <TabList customClass="config-endpoint--tabs">
-            {tabs.map((t, i) => <Tab key={tabs[i].type}>{tabs[i].type}</Tab>)}
+            {_.reduce(
+              configSections,
```

Comment:
Javascript's `reduce` should work just fine here.  Also, storing `supportedConfigs[k]` in a variable would be more efficient.